docassemble/ALToolbox/al_income.py

```diff
@@ -661,20 +673,58 @@ class ALItemizedValue(DAObject):
         when calculating totals.
     """
 
+    def fields(self, use_exists=True) -> List[Dict[str, Any]]:
```

Comment:
Took me a second to figure out this was returning a list for purposes of the YAML `fields` specifier.

In the Weaver, our pattern is `ask_` when what the function returns is a `fields` list for use in YAML--maybe a similar change to the name here would make sense?

Reply:
Are you talking about [user_ask_about_field](https://github.com/SuffolkLITLab/docassemble-ALWeaver/blob/e866c3ee0146638f63a9e2cee130ff533446f2ef/docassemble/ALWeaver/interview_generator.py#L455)? I couldn't find any other instances of `ask` in methods. I copied that method name from AssemblyLine general, like [ALAddress](https://github.com/SuffolkLITLab/docassemble-AssemblyLine/blob/71157d4cce4956e7350a8ebf971375f8ce76bd70/docassemble/AssemblyLine/al_general.py#L72) and [ALIndividual](https://github.com/SuffolkLITLab/docassemble-AssemblyLine/blob/71157d4cce4956e7350a8ebf971375f8ce76bd70/docassemble/AssemblyLine/al_general.py#L545).

I can document thing more for sure. Some clearer type aliases would probably help.

Reply:
`income_fields()` would match the ALGeneral pattern, that works too. And yeah just a docstring would be fine also